sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/PostgresTestOptions.java

```diff
@@ -0,0 +1,40 @@
+package org.apache.beam.sdk.io.jdbc;
+
+import org.apache.beam.sdk.options.Default;
+import org.apache.beam.sdk.options.Description;
+import org.apache.beam.sdk.testing.TestPipelineOptions;
+
+/**
+ * These options can be used by a test connecting to a postgres database to configure their connection.
+ */
+public interface PostgresTestOptions extends TestPipelineOptions {
+    @Description("IP for postgres server")
+    @Default.String("postgres-ip")
+    String getPostgresIp();
+    void setPostgresIp(String value);
+
+    @Description("Username for postgres server")
+    @Default.String("postgres-username")
+    String getPostgresUsername();
+    void setPostgresUsername(String value);
+
+    @Description("Password for postgres server")
+    @Default.String("postgres-password")
+    String getPostgresPassword();
```

Comment:
TODO: This is not an secure authentication strategy. At minimum, this should explicitly be noted as not a best practice.